dunwoody_disaster/animations/RoomAnimation.py

```diff
@@ -7,21 +7,26 @@
 class RoomAnimation(PygameAnimation):
     def __init__(self, background: str, player: str, enemy: str):
         super().__init__()
```

Comment:
@instMeyFTW you can pass in a `size` for the animation widget here. Default size is (666, 360), because that's what the old maps were. @EngineeringDude123 are the new splash backgrounds for the maps a standard size?